elfpy/data/db_schema.py

```diff
@@ -0,0 +1,154 @@
+    gasUsed: Mapped[int | None] = mapped_column(Numeric, default=None)
+
+    #### Fields from solidity function calls ####
+    # These fields map solidity function calls and their corresponding arguments
+    # The params list is exhaustive against all possible methods
+    input_method: Mapped[str | None] = mapped_column(String, default=None)
+
+    # Method: initialize
+    input_params_contribution: Mapped[float | None] = mapped_column(Numeric, default=None)
+    input_params_apr: Mapped[float | None] = mapped_column(Numeric, default=None)
+    input_params_destination: Mapped[str | None] = mapped_column(String, default=None)
+    input_params_asUnderlying: Mapped[bool | None] = mapped_column(Boolean, default=None)
+
+    # Method: openLong
+    input_params_baseAmount: Mapped[float | None] = mapped_column(Numeric, default=None)
+    input_params_minOutput: Mapped[float | None] = mapped_column(Numeric, default=None)
+    # input_params_destination
+    # input_params_asUnderlying
+
+    # Method: openShort
+    input_params_bondAmount: Mapped[float | None] = mapped_column(Numeric, default=None)
+    input_params_maxDeposit: Mapped[float | None] = mapped_column(Numeric, default=None)
+    # input_params_destination
+    # input_params_asUnderlying
+
+    # Method: closeLong
+    input_params_maturityTime: Mapped[int | None] = mapped_column(Numeric, default=None)
+    # input_params_bondAmount
+    # input_params_minOutput
+    # input_params_destination
+    # input_params_asUnderlying
+
+    # Method: closeShort
+    # input_params_maturityTime
+    # input_params_bondAmount
+    # input_params_minOutput
+    # input_params_destination
+    # input_params_asUnderlying
+
+    # Method: addLiquidity
+    # input_params_contribution
+    input_params_minApr: Mapped[float | None] = mapped_column(Numeric, default=None)
+    input_params_maxApr: Mapped[float | None] = mapped_column(Numeric, default=None)
+    # input_params_destination
+    # input_params_asUnderlying
+
+    # Method: removeLiquidity
+    input_params_shares: Mapped[float | None] = mapped_column(Numeric, default=None)
+    # input_params_minOutput
+    # input_params_destination
+    # input_params_asUnderlying
+
+    #### Fields from event logs ####
+    # Addresses in event logs
+    event_from: Mapped[str | None] = mapped_column(String, default=None)
+    event_to: Mapped[str | None] = mapped_column(String, default=None)
+    # args_owner
+    # args_spender
+    # args_id
+    event_value: Mapped[float | None] = mapped_column(Numeric, default=None)
+    event_operator: Mapped[str | None] = mapped_column(String, default=None)
+    event_id: Mapped[int | None] = mapped_column(Numeric, default=None)
+    # Fields calculated from base
+    event_prefix: Mapped[int | None] = mapped_column(Integer, default=None)
+    event_maturity_time: Mapped[int | None] = mapped_column(Numeric, default=None)
+
+    # Fields not used by postprocessing
```

Comment:
Could move these into the docstring. But tbh I feel like we should just get everything and decide later (in postprocessing) if we need it. What's the computational expense for adding these columns? Is the number of columns something we need to be concerned about in general going forward?

Reply:
I'd argue that if the docstrings turn into web docs, I'd keep it where it is now